### PR TITLE
Add climate trace tool page examples

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/templates/custom_dc/climate_trace/map_examples.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "preposition": "across",
+    "examples": [
+      {
+        "text": "countries in Earth",
+        "url": "#%26sv%3DAnnual_Emissions_GreenhouseGas_ForestFire%26pc%3D0%26denom%3DCount_Person%26pd%3DEarth%26ept%3DCountry%26ppt%3DEpaReportingFacility"
+      }
+    ]
+  },
+  {
+    "header": "CO2 from agriculture",
```
Comment (pradh) [inline server/templates/custom_dc/climate_trace/map_examples.json:13]: CO2 emissions? here and below

File: server/templates/custom_dc/climate_trace/scatter_examples.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "header": "Life expectancy vs Oil and Gas emissions per capita",
```
Comment (pradh) [inline server/templates/custom_dc/climate_trace/scatter_examples.json:3]: `vs.`

File: server/templates/custom_dc/climate_trace/scatter_examples.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "preposition": "for countries in",
+    "examples": [
+      {
+        "text": "Earth",
+        "url": "#%26svx%3DAnnual_Emissions_CarbonDioxide_FuelCombustionForDomesticAviation%26lx%3D1%26pcx%3D1%26dx%3DCount_Person%26svy%3DCount_Person_25OrMoreYears_BachelorsDegreeOrHigher_AsFractionOf_Count_Person_25OrMoreYears%26dy%3DCount_Person%26epd%3DEarth%26ept%3DCountry%26dd%3D1"
+      }
+    ]
+  },
+  {
+    "header": "Fertility rate vs CO2 emissions for Power (per capita)",
```
Comment (pradh) [inline server/templates/custom_dc/climate_trace/scatter_examples.json:23]: from Power sector

File: server/templates/custom_dc/climate_trace/timeline_examples.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "header": "CO2 in Manufacturing",
```
Comment (pradh) [inline server/templates/custom_dc/climate_trace/timeline_examples.json:3]: CO2 emissions (and below)

File: server/templates/custom_dc/climate_trace/timeline_examples.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    "preposition": "for",
+    "examples": [
+      {
+        "text": "Japan, China, US, Germany",
+        "url": "#place=country%2FDEU%2Ccountry%2FGBR%2Ccountry%2FJPN%2Ccountry%2FCHN&statsVar=Annual_Emissions_CarbonDioxide_SteelManufacturing&chart=%7B%22amount%22%3A%7B%22pc%22%3Atrue%7D%7D"
+      }
+    ]
+  },
+  {
+    "header": "CO2 (per capita) for Road Transportation",
```
Comment (pradh) [inline server/templates/custom_dc/climate_trace/timeline_examples.json:23]: put per capita at the end of the SV (here and below)

**Review by pradh — APPROVED** [review]
Assuming the "header" field is displayed to user, here are some comments.

File: server/templates/custom_dc/climate_trace/map_examples.json (hunk 1 above)
Comment (shifucun) [reply server/templates/custom_dc/climate_trace/map_examples.json]: Done

File: server/templates/custom_dc/climate_trace/scatter_examples.json (hunk 2 above)
Comment (shifucun) [reply server/templates/custom_dc/climate_trace/scatter_examples.json]: Done

File: server/templates/custom_dc/climate_trace/scatter_examples.json (hunk 3 above)
Comment (shifucun) [reply server/templates/custom_dc/climate_trace/scatter_examples.json]: Done

File: server/templates/custom_dc/climate_trace/timeline_examples.json (hunk 4 above)
Comment (shifucun) [reply server/templates/custom_dc/climate_trace/timeline_examples.json]: Done

File: server/templates/custom_dc/climate_trace/timeline_examples.json (hunk 5 above)
Comment (shifucun) [reply server/templates/custom_dc/climate_trace/timeline_examples.json]: Done

**Pull request merged**